.gitignore

```diff
@@ -46,6 +46,8 @@ coverage.xml
 pytest.xml
 pytest_session.txt
 cover/
+# Ignore ssapy_test_plots folder in tests subfolder
```

Comment:
Please add a newline before this comment.

Reply:
Is this folder created dynamically or will users need to create it on their own?

Reply:
created dynamically